src/test/java/com/cronutils/model/time/ExecutionTimeQuartzWithDayOfYearExtensionIntegrationTest.java

```diff
@@ -68,98 +73,116 @@ public void testNextExecutionEveryTwoWeeksStartingWithFirstDayOfYearIssue249() {
     public void testQuartzCompatibilityIfDoYisQuestionMark() {
-        ExecutionTime executionTime = ExecutionTime.forCron(parser.parse(WITHOUT_SPECIFIC_DAY_OF_YEAR));
-        ExecutionTime quartzExecutionTime = ExecutionTime.forCron(quartzParser.parse(WITHOUT_DAY_OF_YEAR));
-        
-        ZonedDateTime start = ZonedDateTime.of(2017, 1, 1, 0, 0, 0, 0, UTC).minusSeconds(1); 
+        final ExecutionTime executionTime = ExecutionTime.forCron(parser.parse(WITHOUT_SPECIFIC_DAY_OF_YEAR));
+        final ExecutionTime quartzExecutionTime = ExecutionTime.forCron(quartzParser.parse(WITHOUT_DAY_OF_YEAR));
+
+        ZonedDateTime start = ZonedDateTime.of(2017, 1, 1, 0, 0, 0, 0, UTC).minusSeconds(1);
         for (int i = 0; i < 12; i++) {
-            ZonedDateTime expectedDateTime = quartzExecutionTime.nextExecution(start).get();
+            final ZonedDateTime expectedDateTime = quartzExecutionTime.nextExecution(start).get();
             assertEquals(quartzExecutionTime.nextExecution(start).get(), executionTime.nextExecution(start).get());
             start = expectedDateTime.plusSeconds(1);
         }
     }
-    
+
+
     @Test //issue #190
     public void testExecutionTimesWithIncrementsGreaterThanDaysOfMonth() {
         final int increment = 56;
         final String incrementGreaterDaysOfMonthStartingWithFirstDayOfYear = "0 0 0 ? * ? * 1/" + String.valueOf(increment);
-        ExecutionTime executionTime = ExecutionTime.forCron(parser.parse(incrementGreaterDaysOfMonthStartingWithFirstDayOfYear));
-        ZonedDateTime start = ZonedDateTime.of(2017, 1, 1, 0, 0, 0, 0, UTC); 
+        final ExecutionTime executionTime = ExecutionTime.forCron(parser.parse(incrementGreaterDaysOfMonthStartingWithFirstDayOfYear));
+        ZonedDateTime start = ZonedDateTime.of(2017, 1, 1, 0, 0, 0, 0, UTC);
         for (int i = 0; i < 6; i++) {
-            ZonedDateTime expected = start;
-            ZonedDateTime actual = executionTime.nextExecution(start.minusSeconds(1)).get();
+            final ZonedDateTime expected = start;
+            final ZonedDateTime actual = executionTime.nextExecution(start.minusSeconds(1)).get();
             assertEquals(expected, actual);
             start = expected.plusDays(increment);
         }
     }
-    
-    private static ZonedDateTime truncateToDays(ZonedDateTime dateTime){
+
+
+    @Test
+    public void testCustomQuarterlySchedule() {
+        final ZonedDateTime firstDayOf2016 = ZonedDateTime.of(2016, 1, 1, 0, 0, 0, 0, UTC);
+        final ZonedDateTime startWithLastDayOf2015 = ZonedDateTime.of(2015, 12, 31, 0, 0, 0, 0, UTC);
+        final String customQuaterlyStartingWithDay47OfYear = "0 0 0 ? * ? * 47/91";
+        final ExecutionTime executionTime = ExecutionTime.forCron(parser.parse(customQuaterlyStartingWithDay47OfYear));
+        final ZonedDateTime[] expectedExecutionTimes = IntStream.range(0, 4).mapToObj(i -> firstDayOf2016.withDayOfYear(47 + i * 91)).toArray(ZonedDateTime[]::new);

```

Comment:
This line is causing the build to fail. 
It should be Java 1.7 compatible.

Reply:
@agorton Thanks! Will fix ASAP and merge the PR! Thanks!